blog/_posts/2017-10-3-RGSoC- the grand finale.md

```diff
@@ -0,0 +1,99 @@
+---
+title: RGSoC- the Grand Finale.
+layout: post
+created_at: Tue October 3 2017
```

Comment:
Oops, @Anagha22. You ended up adding whole new file. Please remove it. And also consider doing changes I've suggested below in your original file.